CONTRIBUTING.md

```diff
@@ -0,0 +1,168 @@
+# How to contribute
+
+## Prerequisites
+
+- [Node.js][n] 16+;
+- [Git][g];
+- [Docker Desktop][d];
+- IDE:
+  - JetBrains — any with the [Node.js plugin][np], as example [Webstorm][ws];
+  - or Microsoft [Visual Studio Code][vsc];
```

Comment:
Any particular reason we would want to list IDEs here in the prerequisites?  It seems to me like developers should be able to use whatever they like.

Reply:
I can remove this item from prerequisites list.

Reply:
Fixed.